jakarta-eclipse/src/org/jakarta/jdt/servlet/CompleteFilterAnnotationQuickFix.java

```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************
+* Copyright (c) 2020, 2021 IBM Corporation and others.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+* which is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+*
+* Contributors:
+*     IBM Corporation - initial API and implementation
+*******************************************************************************/
+
+package org.jakarta.jdt.servlet;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.jdt.core.dom.IBinding;
+import org.eclipse.lsp4j.CodeAction;
+import org.eclipse.lsp4j.Diagnostic;
+import org.jakarta.codeAction.JavaCodeActionContext;
+import org.jakarta.codeAction.proposal.ChangeCorrectionProposal;
+import org.jakarta.codeAction.proposal.ModifyAnnotationProposal;
+
+/**
+ * QuickFix for fixing {@link ServletConstants#DIAGNOSTIC_CODE_FILTER_MISSING_ATTRIBUTE} error
+ * and {@link ServletConstants#DIAGNOSTIC_CODE_FILTER_MISSING_ATTRIBUTE} error
+ * by providing several code actions:
+ *
+ *
+ * {@link ServletConstants#DIAGNOSTIC_CODE_FILTER_MISSING_ATTRIBUTE}
+ * <ul>
+ * <li> Add the `value` attribute to the `@WebFilter` annotation
+ * <li> Add the `urlPatterns` attribute to the `@WebFilter` annotation
+ * <li> Add the `servletNames` attribute to the `@WebFilter` annotation
+ * </ul>
+ *
+ * {@link ServletConstants#DIAGNOSTIC_CODE_FILTER_MISSING_ATTRIBUTE}
+ * <ul>
+ * <li> Remove the `value` attribute from the `@WebFilter` annotation
+ * <li> Remove the `urlPatterns` attribute from the `@WebFilter` annotation
+ * <li> Remove the `servletNames` attribute from the `@WebFilter` annotation
```

Comment:
I don't believe we have a quick to remove the `servletNames` attribute so we should remove this, correct me if I'm wrong.